server/routes/varyingLicences/handlers/caseload.ts

```diff
@@ -79,4 +80,14 @@ export default class CaseloadRoutes {
       search,
     })
   }
+
+  getReviewNeededCasesToTop(a: { licenceStatus: LicenceStatus }, b: { licenceStatus: LicenceStatus }) {
```

Comment:
Maybe rename to something like `prioritiseReviewNeeded`? `get` implies a `GET` request